src/howler.core.js

```diff
@@ -2118,6 +2119,7 @@
           self.load();
         }
       };
+      xhr.onprogress = self._xhronprogress;
```

Comment:
I think it would be better to use the event system already in place, for example: 

```
xhr.onprogress = function(e) {
    self._emit('downloading', e);
}
```